packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/InstanceManager.java

```diff
@@ -192,25 +175,23 @@ public <T> T getInstance(long identifier) {
    * Returns whether this manager contains the given `instance`.
    *
    * @param instance the instance whose presence in this manager is to be tested.
-   * @return whether this manager contains the given `instance`. If the manager is closed, returns
-   *     `false`.
+   * @return whether this manager contains the given `instance`.
    */
   public boolean containsInstance(@Nullable Object instance) {
-    if (assertNotClosed()) {
-      return false;
-    }
+    logWarningIfFinalizationListenerHasStopped();
     return identifiers.containsKey(instance);
   }
 
   /**
-   * Closes the manager and releases resources.
+   * Stop the periodic run of the {@link FinalizationListener} for instances that have been garbage
+   * collected.
    *
-   * <p>Methods called after this one will be ignored and log a warning.
+   * <p>The InstanceManager can continue to be used, but the {@link FinalizationListener} will no
+   * longer be called and methods will log a warning.
    */
-  public void close() {
+  public void stopFinalizationListener() {
     handler.removeCallbacks(this::releaseAllFinalizedInstances);
-    isClosed = true;
-    clear();
+    hasFinalizationListenerStopped = true;
```

Comment:
Is it possible to derive this value instead of using a boolean? 
For example if handler.removeCallbacks does not have any more callbacks?

Reply:
It looks like there is a [Handler.hasCallbacks](https://developer.android.com/reference/android/os/Handler#hasCallbacks(java.lang.Runnable)), but it requires Android 29+ unfortunately.